### PR TITLE
Jasondev

### PR DESCRIPTION
**Pull request by xuesongc4**
sign up page done

### CODE REVIEW

File: pages/leader_board.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div ng-controller="leaderboard as lb">
     <div class="league_landing">
         <div class="league_button_history">
-            <img src="images/Trophy.png"></img> Leader Board
+            <img src="../images/Trophy.png"></img> Leader Board
```
Comment (kylemarx86) [inline pages/leader_board.html]: This change doesn't  seem to work on my local. All the other images seem to work without the leading '../'

File: pages/client_login.php [hunk 2]
```diff
@@ -60,20 +37,96 @@
...
         </div>
         <div class="sign_up_menu">
-            <input class="input" type="text" name="username_signup" placeholder="Username" id="username" onblur="checkUsername($('#username').val())"><br>
-            <input class="input" type="text" name="email_signup" placeholder="Email"><br>
-            <input class="input" type="password" name="password_signup" placeholder="Password"><br>
-            <input class="input" type="password" name="password_signup_confirm" placeholder="Confirm Password"><br>
-            <button class="user_login" name="user_login">
+            <input class="input_signup" type="text" name="username_signup" placeholder="Username" id="username" onblur="checkUsername()"><span class="warning">Username Taken</span><br>
+            <input class="input_signup" type="text" name="email_signup" placeholder="Email"><br>
+            <input id='password'class="input_signup" type="text" name="password_signup" placeholder="Password" onblur="checkPass()"><br>
```
Comment (kylemarx86) [inline pages/client_login.php]: Would it be better to have type password for these or did you and Dan decide this was better?

**Review by kylemarx86 — CHANGES_REQUESTED** [review]

File: pages/client_login.php (hunk 2 above)
Comment (xuesongc4) [reply pages/client_login.php]: it should be password, i just changed it to text for testing purposes and forgot to change it back, updating now

File: pages/leader_board.html (hunk 1 above)
Comment (xuesongc4) [reply pages/leader_board.html]: green one should be correct with the leaderboard file in the pages folder, you have to go up 1 directory into the images folder to get to trophy.png

**Pull request merged**